include/classes/HeyPublisher/Page.class.php

```diff
@@ -193,16 +206,19 @@ protected function validate_nonced_field() {
   // @since 2.8.0
   // Get the form action url as a relative url
   // Replaces function of same name in HeyPublisher class
+  // @deprecated  3.3.0
+  // Change all calls to this function to use `nonced_url`
   protected function get_form_url_for_page($action=null,$delete=null) {
```

Comment:
`grep`ing the code, I can't find any call to this function.  It should be removed and commentary made in the changelog as to why.

Reply:
resolved by e7ccb89